packages/map-template/src/components/ExternalIds/ExternalIds.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import './ExternalIds.scss';
+import { ReactComponent as CloseIcon } from '../../assets/close.svg';
+import ListItemLocation from "../WebComponentWrappers/ListItemLocation/ListItemLocation";
+
+/**
+ * Show list of locations filtered based on external ids.
```

Comment:
Could we call it ExternalIDsList then? 

Or maybe just LocationsList? There's nothing in here that requires the locations to be based on external ID's (the onLocationsFilteredByExternalIds could be moved outside the component).

Reply:
Are you referring to the ExternalID function on line 17, to rename it? It's a bit difficult to see in GitHub which lines you highlighted.

Reply:
Yes, the name of the component.

Reply:
Renamed the function and made sure it's imported correctly.